pkgs/data/fonts/borg-sans-mono/default.nix

```
@@ -0,0 +1,24 @@
+{ lib, fetchzip }:
+
+let
+  version = "0.2.0";
+in
+fetchzip {
+  name = "borg-sans-mono-ttf-${version}";
```

I like to have `pname` and `version` up here a bit like a normal drv and then join them below in name

`name = "${pname}-${version}";`

```suggestion
let
  pname = "borg-sans-mono-ttf";
  version = "0.2.0";
in
fetchzip {
  name = "${pname}-${version}";
```